tests/fv3_conf/cpld_datm_gefs.IN

```diff
@@ -22,4 +22,7 @@ if [[ $OCNRES == '025' ]]; then
 else
   cp    @[INPUTDATA_ROOT]/MOM6_IC/@[OCNRES]/2011100100/MOM6_IC_TS*.nc ./INPUT/MOM6_IC_TS.nc
   cp    @[INPUTDATA_ROOT]/CICE_IC/@[OCNRES]/cice_model_@[ICERES].cpc*.nc cice_model.res.nc
+  # MOM6 increment file
+  #cp    @[INPUTDATA_ROOT]/MOM6_IC/@[OCNRES]/2011100100/mom6_increment.nc ./INPUT/mom6_increment.nc
+  cp    /scratch2/BMC/gsienkf/Philip.Pegion/MOM6-IAU/mom6_increment.nc ./INPUT/mom6_increment.nc
```

Comment:
Please revert this change and test on hera.

Reply:
testing now